.ci/pytorch/win-test-helpers/installation-helpers/install_sccache.py

```diff
@@ -0,0 +1,46 @@
+import os
+import subprocess
+
+
+tmp_dir_win = os.environ['TMP_DIR_WIN']
+directory = tmp_dir_win + '\\bin'
+os.mkdir(directory, mode=0o666)
+
+if 'REBUILD' not in os.environ:
+
+    while True:
+
+        try:
+            result = subprocess.run(tmp_dir_win + '\\bin\\sccache.exe --show-stats', shell=True)
+            result.check_returncode()
```

Comment:
When called immediately after `subprocess.run` the `check_returncode()` [method](https://docs.python.org/3/library/subprocess.html#subprocess.CompletedProcess.check_returncode) behaves the same as using the [check variable](https://docs.python.org/3/library/subprocess.html#using-the-subprocess-module) `subprocess.run(.....check=True)`.

Reply:
Change this for I THINK*** all occurrences of `check_returncode()`